next.config.js

```diff
@@ -14,6 +14,57 @@ const regexEqual = (x, y) => {
   );
 };
 
+const ContentSecurityPolicy = `
+  default-src 'self';
+  connect-src 'self' https://variant.innocraft.cloud/;
+  script-src 'self' 'sha256-j6xN8x073Dhm+Ee4HKwIIRXsHIqI5aIRHC0pgnhVcJY=' https://variant.innocraft.cloud/ ${
```

Comment:
Should have used 'strict-dynamic' but there is no good way of using generated nonces in Nextjs https://github.com/vercel/next.js/discussions/21587